docs/api/native-image.md

```diff
@@ -198,6 +198,12 @@ The `hslShift` is applied to the image with the following rules
 This means that `[-1, 0, 1]` will make the image completely white and
 `[-1, 1, 0]` will make the image completely black.
 
+In some cases, the `NSImageName` doesn't match its string representation; one example of this is `NSFolderImageName`, whose string representation would actually be `NSFolder`. Therefore, you'll need to determine the correct string representation for your image before passing it in. This can be done with the following:
+
+`echo -e '#import <Cocoa/Cocoa.h>\nint main() { NSLog(@"%@", SYSTEM_IMAGE_NAME); }' | clang -otest -x objective-c -framework Cocoa - && ./test`
```

Comment:
That's one heck of a one-liner 😆

Reply:
maybe also note that you should replace `SYSTEM_IMAGE_NAME` by the standard name you want to look up.